### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by payasbhrigu**
Completed Assignment 2 for week 2 - Transitions.
@anilrao316

### CODE REVIEW

**Comment by payasbhrigu** [comment]
Working Demo:
https://user-images.githubusercontent.com/28836898/169651405-ac5c8347-cdce-4e01-8f3e-e6fc605df6d1.mov

**Comment by payasbhrigu** [comment]
Assignment 2 screenshots:
image
image

With smaller screen:
image

File: Week-2/Assignment2/home.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Transform-Assignment</title>
+</head>
+<body>
+    <div class="main-container">
+        <div class="square-row">
+            <div class="squares red" style="background-color: red;"></div>
```
Comment (anilrao316) [inline Week-2/Assignment2/home.html:12]: can add these as part of external css.. check below as well

**Comment by anilrao316** [comment]
rest looks good to me nicely done appreciate it!

**Comment by payasbhrigu** [comment]
@anilrao316 Thanks for the comments :)
Made the changes as per the comments.

**Pull request merged**